usaspending_api/api_contracts/contracts/v2/agency/awards/count.md

```diff
@@ -0,0 +1,105 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Count of awards for Agencies [/api/v2/agency/awards/count{?fiscal_year,group}]
+
+Return the count of Awards under Agencies
```

Comment:
Should call out that the Awards are grouped by award types. Maybe something like

```
Returns the count of Awards grouped by Award Type under Agencies
```

Reply:
Addressed